src/main/java/com/jagrosh/jmusicbot/commands/dj/JoinCmd.java

```diff
@@ -0,0 +1,31 @@
+package com.jagrosh.jmusicbot.commands.dj;
+
+import com.jagrosh.jdautilities.command.CommandEvent;
+import com.jagrosh.jmusicbot.Bot;
+import com.jagrosh.jmusicbot.commands.DJCommand;
+import net.dv8tion.jda.api.entities.Member;
+import net.dv8tion.jda.api.entities.VoiceChannel;
+import net.dv8tion.jda.api.managers.AudioManager;
+
+public class JoinCmd extends DJCommand {
+
+    public JoinCmd(Bot bot)
+    {
+        super(bot);
+        this.name = "join";
+        this.help = "moves the bot to your channel";
```

Comment:
I think this command should also support joining the bot to a channel by name as well (defaulting to the channel with the DJ if the arguments are left empty). For example, running `join Music` should make it join a channel matching 'Music' even if the DJ is not in that channel.
```suggestion
        this.help = "makes the bot join a voice channel";
        this.arguments = "[channel]";
```